test/test_autograd.py

```diff
@@ -6709,6 +6709,43 @@ def test_GRU_grad_and_gradgrad(self, device):
             mod = torch.nn.GRU(hsize, hsize, bias=bias).to(device).to(torch.float64)
             self._test_rnn_mod(mod, inp)
 
+    def test_copysign_grad(self, device):
```

Comment:
nit: maybe a better name is `test_copysign_edge_cases` or `test_copysign_subgradient` to differentiate it from the other copysign gradient tests (the ones autogenerated by common method invocations)